src/Platform/Sqlite.php

```diff
@@ -43,7 +42,7 @@ public function quoteValue(string $value): string
             $resource = $resource->getConnection()->getResource();
         }
 
-        if ($resource instanceof \PDO) {
+        if ($resource instanceof PDO) {
```

Comment:
When this is changed the import for *\Pdo\Pdo has to change with it or it causes issues for devs in windows.

Reply:
I can handle the \PDO to PDO changes. It will be easier for me since I dev on windows. The code will still run (in most cases) but the DX will be extremely degraded since one or the other will show as a "class not found" or whatever their particular ide displays for that. Plus, if there is a typo in anything that is already showing as a false positive its really really hard to find since the ide will not show it to you.

I'll review it here shortly.